admin/spec/mocks/check-window.js

```diff
@@ -1,12 +1,15 @@
 const mongoose = require('mongoose')
 mongoose.Promise = global.Promise
 const moment = require('moment')
+const {Types: {ObjectId}} = require('mongoose')
```

Comment:
makes sense in this context (only one usage) but please import entire objects by default